oshi-core/src/main/java/oshi/hardware/common/AbstractVirtualMemory.java

```diff
@@ -39,4 +40,15 @@
     protected long swapUsed = -1L;
     protected long swapPagesIn = -1L;
     protected long swapPagesOut = -1L;
+    
+    @Override
+    public String toString() {
+    	StringBuilder sb = new StringBuilder();
+    	sb.append("Swaps used: ");
```

Comment:
Similar to the memory header, let's put "Swap: " in the SystemInfo test and just put "Used:" here instead.